pages/transfer.vue

```diff
@@ -200,6 +200,18 @@ export default class Transfer extends mixins(
     return this.unit === 'KSM'
   }
 
+  get isBSX(): boolean {
+    return this.unit === 'BSX'
+  }
+
+  get chainPrefix() {
+    if (this.isBSX) {
+      return 'bsx'
+    } else {
+      return 'rmrk'
+    }
+  }
+
```

Comment:
Why :/


This brings a lot of breaking changes

Reply:
took the name too literally, since there's no prefix https://beta.kodadot.xyz/transfer

Reply:
you still know the current prefix by prefixmixin

Reply:
now I know